lagrangebench/train/trainer.py

```diff
@@ -302,6 +302,9 @@ def _train(
                 # unroll for push-forward steps
                 _current_pos = raw_batch[0][:, :, :input_seq_length]
                 for _ in range(unroll_steps):
+                    if neighbors_batch.did_buffer_overflow.sum() > 0:
+                        print(f"Overflowed neighbor list at push-forward, step {step}")
+                        break
```

Comment:
Is it better to break immediately or to let it go through with the normal overflow handling? Past overflows are kept in the neighbor list error code until they get fixed.